AD419.DataHelper.Web/Views/ConsolidationCodes/Edit.cshtml

```diff
@@ -6,16 +6,22 @@
 
 <h2>Edit</h2>
 
-
 @using (Html.BeginForm())
 {
     @Html.AntiForgeryToken()
     
     <div class="form-horizontal">
-        <h4>ConsolidationCodes</h4>
+        <h4>Consolidation Codes</h4>
         <hr />
         @Html.ValidationSummary(true, "", new { @class = "text-danger" })
-        @Html.HiddenFor(model => model.ObjectConsolidationNumber)
+        
+        <div class="form-group">
+            @Html.LabelFor(model => model.ObjectConsolidationNumber, htmlAttributes: new { @class = "control-label col-md-2" })

```

Comment:
@taylorkj I don't think col-md-2 does anything here, that's for <div> and the grid system, not labels.  Probably doesn't hurt either but just an FYI that's not how the col-* stuff usually works